build-support/flake8/.flake8

```diff
@@ -0,0 +1,22 @@
+[flake8]
+extend-ignore:
+  E203,  # whitespace before ':' (conflicts with Black)
+  E231,  # Missing whitespace after `,` (enable once fixed)
+  E262,  # inline comment doesn't start with `#` (enable once fixed)
+  E266,  # Too many leading # for block comment (maybe enable once fixed)
+  E301,  # expected 1 blank line (enable once fixed)
+  E306,  # expected 1 blank line (enable once fixed)
+  E501,  # line too long (> 79 characters)
+  E711,  # Comparison to `None` is not idiomatic (enable once fixed)
+  E712,  # Comparison to `False` is not idiomatic (enable once fixed)
+  E713,  # Test for membership should be `not in`  (enable once fixed)
+  E731,  # Do not assign a lambda expression
+  E741,  # Ambiguous variable name (enable once fixed)
+  F723,  # Unrecognized syntax in type comment (remove on next Flake8 release)
+  F821,  # undefined name (remove on next Flake8 release)
```

Comment:
See https://github.com/PyCQA/pyflakes/issues/475. This was fixed back in August 2019 but Pyflakes hasn't had any releases since then. They opened a ticket 9 days ago to track the next release: https://github.com/PyCQA/pyflakes/issues/520.